src/nucleic_acid/tn93/absolute.jl

```diff
@@ -71,13 +71,13 @@ end
   e₃ = exp(-(πY * α₁ + πR * β) * t)
 
   P₁  = πA + (πA * πY / πR) * e₁ + (πG / πR) * e₂
-  P₂  = πC + (πT * πR / πY) * e₁ + (πT / πY) * e₃
+  P₂  = πC + (πC * πR / πY) * e₁ + (πT / πY) * e₃ ####
```

Comment:
Looks good! Just remove the ####'s at the end of lines 74 and 80 here 🙂

Reply:
Done!